util/check-format-commit.sh

```diff
@@ -84,6 +84,11 @@ fi
 # including the line ranges that are changed by the commits
 # It produces output of this format:
 # <file name> <change start line>, <change line count>
+#
+# Note: git diff has this odd behavior in which one line changes
+# sometimes produce a range in which the end line count is omitted
+# correct this by detecting the lack of a comma in the $3 field, and
+# adding a ",0" if its missing
```

Comment:
`git diff` does it according to documented unified diff format, which is a GNU invention.  See https://www.gnu.org/software/diffutils/manual/html_node/Detailed-Unified.html

Reply:
@nhorman How about rebasing to remove [this commit](https://github.com/openssl/openssl/pull/24861/commits/cdca0cd04b6761ece515796b7f19d536f8a8a225) in favour of #24900?

Reply:
sure